src/base/utils/net.cpp

```diff
@@ -38,5 +38,23 @@ namespace Utils
         {
             return !QHostAddress(ip).isNull();
         }
+
+        bool isLoopbackAddress(const QHostAddress &clientAddress)
+        {
+            return clientAddress == QHostAddress::LocalHost
+                    || clientAddress == QHostAddress::LocalHostIPv6
+                    || clientAddress == QHostAddress("::ffff:127.0.0.1");
+        }
+
+        bool isLANAddress(const QHostAddress &clientAddress)
+        {
+            return clientAddress.isInSubnet(QHostAddress::parseSubnet("10.0.0.0/8"))
```

Comment:
>What's the purpose of using QLatin1String in this instance? 

QHostAddress is constructed via `QHostAddress(const QString &address)` and then from http://doc.qt.io/qt-5/qlatin1string.html#details :
>and is faster than converting the Latin-1 strings using QString::fromLatin1().